docs/partners/certification.md

```diff
@@ -0,0 +1,58 @@
+---
+id: certification 
+title: Certification 
+---
+
+Certification is the process by which Cortex XSOAR validates and marks the quality of the Content Pack, free or paid. It correlates with the confidence a customer should have in the delivered content offering. Certified Packs indicate the highest level of quality and reliability in our ecosystem. Palo Alto Networks sets detailed standards for certification but see below for high level guidance.
+
+
+><i>Note:</i> certification is separate from support; all Partner-submitted Packs must be supported by the Partner, regardless if the Pack is certifed or not. 
+
+
+## Levels of Certification
+
+When beginning the development process, the Contributor specifies the certification level they intend to achieve, and this will dictate the tasks the Contributor must complete. 
+
+- Not Certified: The Pack has gone through full security testing and basic functionality testing by Palo Alto Networks.
```

Comment:
I think there was a discussion around this, how we will call non certified, `Verified` or something like this.

Reply:
I thought we pushed that off for now? Since thats product changes too, no @anara123 ?

Reply:
Confirmed its product changes, so we'll pick it up later. I like that word much better.... :)